README.md

```diff
@@ -0,0 +1,10 @@
+# Saftladen
+
+[zaftˈlaːdn̩] Juice store; slightly disrespectful term used to describe a facility with bad organization or poor service
```

Comment:
s/disrespectful/impolite

Reply:
I think "disorganized" catches the meaning of how that term is most commonly used quite well.

Reply:
you guys know better ;) , i'd just rather avoid `disrespectful` word, imo

Reply:
After consulting the dictionary another time, i think "disparaging" has the most fitting german translations. Nuances in language are so hard... 
```suggestion
[zaftˈlaːdn̩] Juice store; disparaging term used to describe a facility with bad organization or poor service
```

Reply:
Rereading the full sentence, perhaps something like:
```
[zaftˈlaːdn̩] Juice store; a term often applied to arbitrary shops or establishments to imply that they are run in a somewhat disorganized or amateurish fashion.
```

Reply:
Sorry, i was a little too fast here... Are you willing to open a PR? Also I am sure there is a lot that can be added to the README from experience once someone actually starts adding new casts.